README.md

```diff
@@ -5,7 +5,7 @@ for Python.
 [![Build Status](https://travis-ci.org/amzn/ion-python.svg?branch=master)](https://travis-ci.org/amzn/ion-python)
 [![Documentation Status](https://readthedocs.org/projects/ion-python/badge/?version=latest)](https://ion-python.readthedocs.io/en/latest/?badge=latest)
 
-This package is designed to work with **Python 3.6+**
+This package is designed to work with **Python 3.6** and above. Sunsetted versions may not be supported once they are 6 months or more past EOL.
```

Comment:
Should we update this to 3.7?

Reply:
Good Question! I _did_ but then I reconsidered. It is still true that it was designed to work with Python 3.6, that hasn't changed. But there is nothing in this change that means it won't work with 3.6, and it's not true that it was designed to work with a more recent Python!

Maybe better would be to change it to:
> This package is designed to work with Python 3. <<short but descriptive statement about range of minor versions supported and api optimized for here that I'm still working on>>

Reply:
Works for me!